neuralprophet/forecaster.py

```diff
@@ -3044,22 +3033,27 @@ def conformalize(self, df_cal, alpha, method="naive", plotting_backend="default"
                     * ``plotly``: Use the plotly backend for plotting
                     * ``matplotlib``: Use matplotlib backend for plotting
                     * ``default`` (default): Use matplotlib backend for plotting
+            kwargs : dict
+                additional predict parameters for test df
         """
-        df_cal = self.predict(df_cal)
+        # conformalize
+        df_cal = self.predict(calibration_df)
```

Comment:
What does `df_cal` stand for? Could we find a more verbose and clear name instead?